From: Debian Science Team <debian-science-maintainers@lists.alioth.debian.org>
Date: Sat, 26 Apr 2014 19:21:05 -0700
Subject: 08_do_not_set_qt_default_term

---
 src/term.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/term.c b/src/term.c
index 7d850c4..6d81bae 100644
--- a/src/term.c
+++ b/src/term.c
@@ -1557,16 +1557,16 @@ init_terminal()
 	    term_name = "sun";
 #endif /* SUN */
 
-#ifdef QTTERM
-	if (term_name == (char *) NULL)
-		term_name = "qt";
-#endif
-
 #ifdef WXWIDGETS
 	if (term_name == (char *) NULL)
 		term_name = "wxt";
 #endif
 
+#ifdef QTTERM
+	if (term_name == (char *) NULL)
+		term_name = "qt";
+#endif
+
 #ifdef _Windows
 	/* let the wxWidgets terminal be the default when available */
 	if (term_name == (char *) NULL)
