From: Debian Science Team <debian-science-maintainers@lists.alioth.debian.org>
Date: Sat, 26 Apr 2014 19:21:05 -0700
Subject: no_plugin_demo

===================================================================
---
 demo/plugin/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/demo/plugin/Makefile.am b/demo/plugin/Makefile.am
index 8432424..dc430fe 100644
--- a/demo/plugin/Makefile.am
+++ b/demo/plugin/Makefile.am
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in -*-Makefile-*-
 AUTOMAKE_OPTIONS = foreign
 
-noinst_PROGRAMS = demo_plugin.so
+noinst_PROGRAMS =
 
 demo_plugin_so_SOURCES = demo_plugin.c
 demo_plugin_so_CFLAGS = -rdynamic -fPIC -I$(top_srcdir)/src
