From: Dima Kogan <dima@secretsauce.net>
Date: Sat, 26 Apr 2014 23:00:34 -0700
Subject: '1' and '2' now control the clipboard mode too

---
 src/mouse.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mouse.c b/src/mouse.c
index 91e4036..a1a6a31 100644
--- a/src/mouse.c
+++ b/src/mouse.c
@@ -1196,6 +1196,7 @@ builtin_decrement_mousemode(struct gp_event_t *ge)
 	return "`builtin-previous-mouse-format`";
     }
     incr_mousemode(-1);
+    incr_clipboardmode(-1);
     return (char *) 0;
 }
 
@@ -1206,6 +1207,7 @@ builtin_increment_mousemode(struct gp_event_t *ge)
 	return "`builtin-next-mouse-format`";
     }
     incr_mousemode(1);
+    incr_clipboardmode(1);
     return (char *) 0;
 }
 
@@ -1215,7 +1217,6 @@ builtin_decrement_clipboardmode(struct gp_event_t *ge)
     if (!ge) {
 	return "`builtin-decrement-clipboardmode`";
     }
-    incr_clipboardmode(-1);
     return (char *) 0;
 }
 
@@ -1225,7 +1226,6 @@ builtin_increment_clipboardmode(struct gp_event_t *ge)
     if (!ge) {
 	return "`builtin-increment-clipboardmode`";
     }
-    incr_clipboardmode(1);
     return (char *) 0;
 }
 
