Index: gnuplot.deb/term/gd.trm
===================================================================
--- gnuplot.deb.orig/term/gd.trm	2013-03-07 14:01:29.881011968 -1000
+++ gnuplot.deb/term/gd.trm	2013-03-07 14:13:13.592501493 -1000
@@ -974,7 +974,7 @@
 	if (external_default)
 		png_state.ttffont = gp_strdup(external_default);
 	else	/* Might as well try some plausible font; it's no worse than failing immediately */
-		png_state.ttffont = gp_strdup("arial");
+		png_state.ttffont = gp_strdup("/usr/share/fonts/truetype/liberation/LiberationSans-Regular.ttf");
 
 	free(png_state.default_ttffont);
 	png_state.default_ttffont = gp_strdup(png_state.ttffont);
