Index: gnuplot.deb/demo/plugin/Makefile.am
===================================================================
--- gnuplot.deb.orig/demo/plugin/Makefile.am	2014-04-14 14:45:12.418128686 -0700
+++ gnuplot.deb/demo/plugin/Makefile.am	2014-04-14 14:45:28.634209189 -0700
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in -*-Makefile-*-
 AUTOMAKE_OPTIONS = foreign
 
-noinst_PROGRAMS = demo_plugin.so
+noinst_PROGRAMS =
 
 demo_plugin_so_SOURCES = demo_plugin.c
 demo_plugin_so_CFLAGS = -rdynamic -fPIC -I$(top_srcdir)/src
